dts/client.py

```diff
@@ -216,23 +250,24 @@ def transferStatus(self, id):
         else:
             results = response.json()
             return TransferStatus(
-                id                    = response['id'],
-                status                = response['status'],
-                message               = response['message'] if 'message' in response else None,
-                num_files             = response['num_files'],
-                num_files_transferred = response['num_files_transferred'],
+                id                    = results['id'],
+                status                = results['status'],
+                message               = results['message'] if 'message' in results else None,
```

Comment:
might be simpler to do `results.get("message")` - it will set `message` to `None` if `results` does not have a `message` key